src/scss/main.scss

```diff
@@ -124,12 +127,16 @@ input:placeholder-shown {
 
 #every-time{
   font-size:18px;
+  font-family:Arial, FontAwesome;
```

Comment:
I think the FontAwesome icons should work without this `FontAwesome`  in the font family. But maybe I'm missing something.